plaso/lib/lexer.py

```diff
@@ -139,18 +151,20 @@ def AddOperands(self, lhs, rhs):
       ParseError: if either left hand side or right hand side expression
           is not an instance of Expression.
     """
-    if isinstance(lhs, Expression) and isinstance(rhs, Expression):
-      self.args = [lhs, rhs]
-    else:
-      raise errors.ParseError(
-          'Expected expression, got {0:s} {1:s} {2:s}'.format(
-              lhs, self.operator, rhs))
+    if not isinstance(lhs, Expression):
+      raise errors.ParseError('Left hand side is not an expression')
```

Comment:
Please add information about what lhs and rhs are to these error messages

Reply:
Not sure what you mean, but let's chat about this in the clean up. Adding a note for now.